submission-scripts/script-templates/magpie-hive

```diff
@@ -0,0 +1,71 @@
+#############################################################################
+##Hive configurations
+#############################################################################
+#
+# Should Hive be run
+#
+# Specify yes or no.  Defaults to no.
+#
+export HIVE_SETUP=yes
+
+# Version
+
+export HIVE_VERSION="HIVEVERSIONDEFAULT"
+
+# Path to your Hive build/binaries
+#
+# This should be accessible on all nodes in your allocation. Typically
+# this is in an NFS mount.
+#
+# Ensure the build matches the Hadoop/HDFS version this will run against.
+#
+export HIVE_HOME="HIVEDIRPREFIX/base/apache-hive-${HIVE_VERSION}-bin"
+
+# Set the port for Hive to listen on
+
+export HIVE_PORT="11112"
+
+# Set HADOOP_HOME (IF NOT DONE ABOVE)
+# Hive uses Hadoop, so:
+#    you must have Hadoop in your path OR
+#    export HADOOP_HOME=<hadoop-install-dir>
+#
+# export HADOOP_HOME="${_PREFIX}/base/hadoop-${HADOOP_VERSION}"
+
+# need ANT_HOME configured for Hive web interface
+export ANT_LIB="HIVEDIRPREFIX/base/apache-ant-1.9.9/lib"
+
+# Path to store data local to each cluster node, typically something
+# in /tmp.  This will store local conf files and log files for your
+# job.  If local scratch space is not available, consider using the
+# MAGPIE_NO_LOCAL_DIR_DIR option.  See README for more details.
+
+export HIVE_LOCAL_DIR="LOCALDIRPREFIX/hive"
+export HIVE_METASTORE_DIR="LUSTREDIRPREFIX/hive/database/postgres/data"
+export HIVE_DEFAULT_METASTORE_DIR="LUSTREDIRPREFIX/hive/metastore"
+
+# Configurations for Postgres, used as a backend for Hive
+export POSTGRES_HOME="POSTGRESDIRPREFIX/base/pgsql"
+export PGPORT=POSTGRESDEFAULTPORT
+export PGDATA="LUSTREDIRPREFIX/hive/database/postgres/data"
```

Comment:
just for consistency, I think this should be "POSTGRES" instead of "PG", b/c PG could be anything.  Also something like "DATA_DIR" I think would be more descriptive than data.

Reply:
The only reason I went with this naming convention is that Postgres looks for PGPORT and PGDATA environment variables during startup.

Reply:
ahh, wasn't aware it was specific to postgres.